examples/url-sync-json/src/components/Counter/recoil.ts

```diff
@@ -0,0 +1,8 @@
+import { number } from '@recoiljs/refine'
+import { syncEffect } from 'recoil-sync'
+import { atomFamilyWithInitialValue } from 'recoil-sync-next'
+
+export const counter = atomFamilyWithInitialValue<number>({
```

Comment:
Please add second type parameter explicitly.

Reply:
I fixed it.